tests/security/openscap/oscap_oval_scanning.pm

```diff
@@ -67,10 +64,10 @@ sub run {
         result="not\ evaluated"/sxx';
 
     validate_script_output "oscap oval eval --results $oval_result oval.xml", sub { $scanning_match };
-    $self->validate_result($oval_result, $result_match);
+    validate_result($oval_result, $result_match);
```

Comment:
Same here, I don't see `$oval_result`

Reply:
These variables were defined in openscaptest module which imported to here. We want to keep these files name not changed, because they can be used for later cases.